controllers/amphoracontroller_controller.go

```diff
@@ -463,7 +515,7 @@ func (r *OctaviaAmphoraControllerReconciler) generateServiceConfigMaps(
 		mariadbv1.MariaDBAccountReadyMessage)
 
 	templateParameters := map[string]interface{}{
-		"DatabaseConnection": fmt.Sprintf("mysql+pymysql://%s:%s@%s/%s",
+		"DatabaseConnection": fmt.Sprintf("mysql+pymysql://%s:%s@%s/%s?read_default_file=/etc/my.cnf",
```

Comment:
maybe you haven't noticed it because it was merged recently, but there's also a 2nd DB connection URL (just bellow, line 524), I think it also requires this change.